app/views/metrics/_tile_with_chart.html.erb

```diff
@@ -1,4 +1,7 @@
-<%= polaris_card(title: t("metrics.tiles.#{tile.handle}", default: tile.handle.to_s.humanize)) do |card| %>
+<%= polaris_card(title: t("metrics.tiles.#{tile.handle}", default: tile.handle.to_s.humanize), classes: 'relative') do |card| %>
+  <div class="loading-overlay hidden Polaris-LegacyCard">
```

Comment:
Is it possible to avoid hard coding a Polaris css class here? It would be great to either:

a) Use a `polaris_card` component instead
or
b) Extract needed CSS from `.Polaris-LegacyCard` into a custom class.